### PR TITLE
Add portaudio v19.7.0

### PR DESCRIPTION
**Pull request by akemimadoka**
## Description
Add portaudio v19.7.0

This version fixed bug https://github.com/PortAudio/portaudio/issues/395, this bug makes macOS Big Sur unable to open audio device

### CODE REVIEW

File: recipes/portaudio/all/conanfile.py [hunk 1]
```diff
@@ -56,110 +51,32 @@ def system_requirements(self):
...
-                if self.settings.os == "Macos":
-                    self.copy(pattern="*.dylib", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-                else:
-                    self.copy(pattern="*.so*", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-            else:
-                self.copy(pattern="*.a", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-
-
+        cmake = self._configure_cmake()
+        cmake.install()
```
Comment (Croydon) [inline recipes/portaudio/all/conanfile.py:73]: Similar question, just to make sure: CMake is now installing all relevant files on all platforms?

**Review by Croydon — CHANGES_REQUESTED** [review]

File: recipes/portaudio/all/conanfile.py [hunk 2]
```diff
@@ -14,27 +14,22 @@ class ConanRecipe(ConanFile):
         "fPIC": [True, False],
         "with_alsa":  [True, False],
         "with_jack":  [True, False],
-        "cpp_bindings": [True, False],
```
[the remark this answers is not in the document]
Comment (akemimadoka) [reply recipes/portaudio/all/conanfile.py:17]: cpp_bindings seems not supported in CMake for now(https://github.com/PortAudio/portaudio/issues/539), so I removed it. if CMake is not good enough, I will change the recipe to use Autotools and add it back.

**Comment by akemimadoka** [comment]
I tested the recipe in Windows 7, Ubuntu 20.04, and macOS Big Sur, it seems CMake is working well but lacks the support for cpp_bindings. Would it be better to change the recipe back and use Autotools?

**Comment by Croydon** [comment]
I don't know, but I assume there was a reason for doing it like that in the first place. What was your motivation to change it to CMake-only?

**Comment by akemimadoka** [comment]
In https://github.com/PortAudio/portaudio/pull/434, they seemed to reach a consensus that they would consider removing support for other build systems, so I think making this recipe CMake-only is helpful for compatibility with later versions

**Pull request closed without merging**